### PR TITLE
Get nearby users API

### PR DESCRIPTION
**Pull request by nikhilleo10**
### Ticket Link
Get nearby users API
---

### Related Links

---

### Description
Get nearby users API
---

### Steps to Reproduce / Test

---

---

### Checklist

- [ ] PR description included
- [ ] `yarn test` passes
- [ ] Tests are [changed or added]
- [ ] Relevant documentation is changed or added (and PR referenced)

### GIF's

---

### CODE REVIEW

File: server/gql/models/vehicles/index.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+import { getQueryFields, TYPE_ATTRIBUTES } from '@server/utils/gqlFieldUtils';
+import { GraphQLDriver } from '../drivers';
+
+const { nodeInterface } = getNode();
+
+export const vehicleFields = {
+  brand: { type: new GraphQLNonNull(GraphQLString) },
+  color: { type: new GraphQLNonNull(GraphQLString) },
+  vehicleNo: { type: new GraphQLNonNull(GraphQLString) },
+  type: { type: new GraphQLNonNull(GraphQLString) },
```
Comment (aseer-ws) [inline server/gql/models/vehicles/index.js]: please use GraphQLEnumType

File: server/gql/models/vehicles/index.js [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+  maxCapacity: { type: new GraphQLNonNull(GraphQLInt) },
+  engineType: { type: new GraphQLNonNull(GraphQLString) },
```
Comment (aseer-ws) [inline server/gql/models/vehicles/index.js]: here as well

File: server/gql/models/vehicles/index.js [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+    id: {
+      type: new GraphQLNonNull(GraphQLInt)
+    }
+  },
+  query: {
+    type: GraphQLVehicle,
+    resolve: resolver(db.vehicleModel, {
+      before: (findOptions, args, ctx) => {
+        findOptions.include = findOptions.include || [];
+        findOptions.include.push({
```
Comment (aseer-ws) [inline server/gql/models/vehicles/index.js:95]: Are we keeping this?

Comment (nikhilleo10) [reply server/gql/models/vehicles/index.js:95]: To query driver in the vehicle it is good to have. So let's keep it.

**Review by aseer-ws — APPROVED** [review]

**Pull request merged**